### PR TITLE
Weather app

### PR DESCRIPTION
**Pull request by raheebwa**
## Features
* [x] Loads data from api
* [x]  Uses async functions / Promises
* [x] UI built with bootstrap

### CODE REVIEW

**Review by mcrd25 — CHANGES_REQUESTED** [review]
# changes Required 🔍 👀 

Hey @raheebwa! 👋 

Great job with this app so far! The code looks good and the repository is very professional! 😄 
However, you are missing one REQUIRED FEATURE! Please read below for further explanation. 👇 

#### Required Changes 
- You are missing the feature that changes the appearance of the app based on weather data. (Color, background image, etc) Please see this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/appearance_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app)

***Please makes changes and submit another review request.
If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)

---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Comment by raheebwa** [comment]
Hey @mcrd25 ,
Please check the app again. The background changes depending on the weather conditon

**Comment by mcrd25** [comment]
hi, @raheebwa I will.
I checked many different cities and no change. Different weather conditions and still the same initial image was shown. I will check using your live demo instead of local version now

File: README.md [hunk 1]
```diff
@@ -1,9 +1,57 @@
...
+
+## Built With
+
+- Javscript, HTML/CSS,
+- Bootstrap,
+- webpack
+
+## Live Demo
+
+[Live Demo Link](https://raheebwa.com/mv-weather-app)
```
Comment (MauricioRobayo) [inline README.md]: This link is broken.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import domMaster from './modules/domMaster';
+import utilities from './modules/utilities';
+
+domMaster.renderForecast();
+
+const searchForm = utilities.getEl('search-form');
+
+searchForm.addEventListener('submit', () => {
+  utilities.getEl('weather-data').innerHTML = '';
```
Comment (MauricioRobayo) [inline src/index.js]: You might want to [prevent the default submit event](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event).

```js
searchForm.addEventListener('submit', (e) => {
    e.preventDefault();
  utilities.getEl('weather-data').innerHTML = '';
```

**Review by MauricioRobayo — CHANGES_REQUESTED** [review]
Hi!

The app looks great, but I am having the same issues as @mcrd25 and no matter which city I type, I always get the same result. Take a look at this:

ezgif-1-ce664c76322f

It seems like the request for the city is made, but then the page reloads again with the default city 🤔.

You are submitting a form, be aware of the default event when you submit a form. You want to avoid the default event.

Required changes:

* [ ] Fix bug when submitting a city.
* [ ] The application makes different requests for the Celsius data and the Fahrenheit data (you can get the data for either unit and use math to convert it to the other unit. That beats the point of the assignment) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/requests_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/)).

Take a look at the [units options](https://openweathermap.org/current#data) on the Open Weather API.

Status: __Request changes__.

**Review by shloch — APPROVED** [review]
accomplished

## GREAT JOB

**Pull request merged**